automatic/selenium-chrome-driver/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,31 @@
+Get-Process -Name chromedriver -ErrorAction SilentlyContinue | Stop-Process
+
+$toolsDir = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
+$seleniumDir = "$(Get-ToolsLocation)\selenium"
+$driverPath = "$seleniumDir\chromedriver.exe"
+
+$parameters = Get-PackageParameters
+
+$packageArgs = @{
+  packageName  = 'selenium-chrome-driver'
+  file         = "$toolsDir\"
+  checksum     = '53c2e1f1b9f2a7571483284ea156958674501e8571cfe34be01282b4152e0770138d9aaef12876f109db9d77c1216362d0576aa40b4f19f039d4b97a15054c98'
+  checksumType = 'sha512'
+  destination  = $seleniumDir
+}
+
+Get-ChocolateyUnzip @packageArgs
+Get-ChildItem $toolsDir\*.zip | ForEach-Object { Remove-Item $_ -ea 0 }
+
+Uninstall-BinFile -Name 'chromedriver'
```

Comment:
Should this `Uninstall-Binfile` call be gated behind the parameter like the `Install-BinFile` call is?

Reply:
Not really needed, if it doesn't exist then the call is a no-op (only some debug messages will be outputted).

Reply:
Fair enough.